ts/ui/menu/MJContextMenu.ts

```diff
@@ -78,6 +86,7 @@ export class MJContextMenu extends ContextMenu.ContextMenu {
                 const semantics = this.findID('Settings', 'semantics');
                 input === 'MathML' ? semantics.disable() : semantics.enable();
                 this.getAnnotationMenu();
+                this.dynamicSubmenus();
```

Comment:
I'm wondering if `getAnnotationMenu()` could be folded into `dynamicSubmenus()` eventually (future release).

Reply:
I had been thinking exactly the same thing, but did not want to play with it for the time being. 
I was also thinking that these types of dynamic menu updates could eventually move into the `contextmenu` package altogether. Added it to the futures list.